### PR TITLE
Updates to HTC exercises

### PR DESCRIPTION
**Pull request by iross**
- Overhauled the content of 2.3 + 2.4. They were in the opposite order last year, so had to iron out a lot of "In the last exercise..." type things
- Minor changes
- Pulled in Jason's changes

### CODE REVIEW

File: docs/materials/htc/part2-ex1-files.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Next, look at the output that appears after the `Sandbox:` line; it is the outpu
 | `_condor_stderr`  | Saved standard error from the job           |
 | `_condor_stdout`  | Saved standard output from the job          |
 | `condor_exec.exe` | The executable, renamed from `sandbox.sh`   |
-| `tmp/`            | A directory in which to put temporary files |
+| `tmp/`, `var/tmp/`| Directories in which to put temporary files |
```
Comment (osg-cat) [inline docs/materials/htc/part2-ex1-files.md:45]: Is `tmp` still there at the top level? If not, just drop it. If so, what is the difference? (Not that the difference needs to go into the table, I’m just asking.)

File: docs/materials/htc/part2-ex1-files.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ for word in sorted(words.keys()):
 1.  Download the input file for the script (263K lines, ~1.4 MB) and save it in your submit directory:
 
         :::console
-        username@learn $ wget http://proxy.chtc.wisc.edu/SQUID/osgschool19/mon-2.1-words.txt
+        username@learn $ wget http://proxy.chtc.wisc.edu/SQUID/osgschool20/mon-2.1-words.txt
```
Comment (osg-cat) [inline docs/materials/htc/part2-ex1-files.md]: Ooh, does this exist?

File: docs/materials/htc/part2-ex1-files.md [hunk 3]
```diff
@@ -98,7 +98,7 @@ for word in sorted(words.keys()):
 1.  Do not forget to add a line to name the input file as the argument to the Python script.
 1.  Submit the job, wait for it to finish, and check the output!
 
-If things do not work the first time, keep trying! At this point in the exercises, we are telling you less and less explicitly how to do steps that you have done before. If you get stuck, ask a neighbor or one of the instructors.
+If things do not work the first time, keep trying! At this point in the exercises, we are telling you less and less explicitly how to do steps that you have done before. If you get stuck, ask for help in the Slack channel.
```
Comment (osg-cat) [inline docs/materials/htc/part2-ex1-files.md]: #general, or should we suggest a specific one like #intro-to-htc?

File: docs/materials/htc/part2-ex1-files.md (hunk 2 above)
Comment (osg-cat) [reply docs/materials/htc/part2-ex1-files.md]: This is probably going to conflict with Jason’s PR now. I may need help here.

File: docs/materials/htc/part2-ex1-files.md (hunk 1 above)
Comment (iross) [reply docs/materials/htc/part2-ex1-files.md:45]: My understanding is that it depends on the config. On CHTC, it looks like tmp and var/tmp are both there. I'm fine not mentioning it, but they're both in the sandbox

File: docs/materials/htc/part2-ex1-files.md (hunk 2 above)
Comment (iross) [reply docs/materials/htc/part2-ex1-files.md]: Yup! Somebody much more clever than I already cp'ed last year's squid stuff over. 

I'll take a look at the conflict in a few.

File: docs/materials/htc/part2-ex1-files.md (hunk 3 above)
Comment (iross) [reply docs/materials/htc/part2-ex1-files.md]: Ah, good point. There's a lot of Slack channels ;) I'll update to #intro-to-htc

File: docs/materials/htc/part2-ex1-files.md (hunk 1 above)
Comment (osg-cat) [reply docs/materials/htc/part2-ex1-files.md:45]: No, leave it in.

**Comment by iross** [comment]
Sorry @osg-cat I meant to have the meatier exercise 2.3/2.4 updates on a separate PR, but I bungled the branching here. So it's different from when you looked it over a few hours ago.

**Comment by osg-cat** [comment]
Yeah, this is kind of a mess now. Maybe @brianhlin can help?

**Comment by iross** [comment]
To be clear, there's nothing here that _shouldn't_ be merged. It's just a bigger delta than the original PR that @osg-cat looked at a few hours ago because of the required 2.3/2.4 updates. 

@osg-cat would you prefer those changes come in a different PR?

**Comment by osg-cat** [comment]
I’ll just try to slog through this one as is, thanks.

**Review by osg-cat — APPROVED** [review]
I can’t tell if the new, big changes actually make sense once you read the documents in order — someone should check all that carefully. But the changes seem like they will be fine, so I am approving and merging.

**Pull request merged**